onnxruntime/core/framework/allocator.cc

```diff
@@ -126,6 +139,24 @@ void* AllocateBufferWithOptions(IAllocator& alloc, size_t size, bool use_reserve
   }
   return alloc.Alloc(size);
 }
+
+size_t GetAlignmentForDevice(const OrtDevice& ort_device) {
+  auto mem_type = ort_device.MemType();
+  if (mem_type == OrtDevice::MemType::CPU_ALIGNED_4K) {
+    return kAlloc4KAlignment;
+  }
+  return kAllocAlignment;
+}
+
+bool IsCpuDeviceWithAllocator(const OrtDevice& ort_device) {
+  if (ort_device.Type() == OrtDevice::CPU) {
+    auto mem_type = ort_device.MemType();
+    return mem_type == OrtDevice::MemType::DEFAULT ||
+           mem_type == OrtDevice::MemType::CPU_ALIGNED_4K;
+  }
+  return false;
+}
```

Comment:
I don't really understand what a cpu device without an allocator would be, but this doesn't appear to be used anywhere.

Reply:
Removed